문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,13 @@
+function solution(s) {
+    var answer = false;
+    let counts = {
+        p: 0,
+        y: 0,
+    };
+    for (var i = 0; i <= s.length; i++) {
```

Comment:
배열 반복할때 s.length 전까지 반복해야지 s.length번째까지 반복하면 안됩니다
```suggestion
    for (let i = 0; i < s.length; i++) {
```